e2e-tests/insurance/cypress/e2e/journeys/account/create/account-blocked/create-account-already-exists-and-is-blocked.spec.js

```diff
@@ -0,0 +1,29 @@
+import { INSURANCE_ROUTES as ROUTES } from '../../../../../../../constants/routes/insurance';
+
+const {
+  ACCOUNT: {
+    SUSPENDED: { EMAIL_SENT },
+  },
+  START,
+} = ROUTES;
+
+context('Insurance - Account - Create account - Create an account which already exists and is suspended', () => {
+  const baseUrl = Cypress.config('baseUrl');
+  const emailSentUrl = `${baseUrl}${EMAIL_SENT}`;
+
+  after(() => {
+    cy.deleteAccount();
+  });
+
+  beforeEach(() => {
+    cy.createAnAccountAndBecomeBlocked({});
+
+    cy.navigateToUrl(START);
+  });
+
+  it(`should redirect to ${emailSentUrl} when creating an account which already exists and is blocked`, () => {
+    cy.completeAndSubmitCreateAccountForm({ navigateToAccountCreationPage: true });
+
+    cy.assertUrl(emailSentUrl);
+  });
+});
```

Comment:
Can we add a new test for also signing in so "create account > become blocked > create account > sign in"?

Reply:
done